src/EditorFeatures/Test2/NamingStyles/NamingStylesTests.vb

```diff
@@ -1,7 +0,0 @@
-﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-Namespace Microsoft.CodeAnalysis.Editor.UnitTests.LanguageServices
-    Public Class NamingStylesTests
-        ' TODO
```

Comment:
wait... do we seriously have no VB tests for thsi feature?  @dpoeschl ??

Reply:
there were no tests... but I added them in a different file (and project)